style.css

```diff
@@ -314,11 +428,144 @@ input:focus:invalid {
   box-shadow: none;
 }
 
-.footer_img {
-  position: center;
-  margin: 80px;
-  color: #0d151e;
-  width: 134px;
-  height: 2px;
-  border-radius: 10px;
+.shape {
+  width: 135px;
+  display: center;
+  margin: 32px 8px 12px 22px;
+}
+
+/* Media Queries section */
+
+@media screen and (min-width: 1440px) {
+  /* navbar */
```

Comment:
- [ ] Use media queries for the Desktop breakpoint in the CSS file. (the value for `min-width` property should be in range of `768px` or wider)